test/gui/shared/steps/steps.py

```diff
@@ -285,17 +194,9 @@ def step(context, receiver, resource, permissions):
     socketConnect.sendCommand("SHARE:" + resource + "\n")
     permissionsList = permissions.split(',')
 
-    test.compare(
-        str(waitForObjectExists(names.scrollArea_sharedWith_QLabel).text), receiver
-    )
-    test.compare(
-        waitForObjectExists(names.scrollArea_permissionsEdit_QCheckBox).checked,
-        ('edit' in permissionsList),
-    )
-    test.compare(
-        waitForObjectExists(names.scrollArea_permissionShare_QCheckBox).checked,
-        ('share' in permissionsList),
-    )
+    shareItem = SharingDialog()
+    snooze(10)
```

Comment:
why is there sleep here?

Reply:
That was added during checking for objects in AUI but now I have removed it.